### PR TITLE
Feature/002 entity framework -> dev

### PR DESCRIPTION
**Pull request by WMinowski**

### CODE REVIEW

**Comment by dmyhun-andersen** [comment]
Better name for branch be **Feature/002-Entity-framework-implementation**
Better name for PR be **Feature/002-Entity-framework-implementation -> dev**

File: MVC-App/Controllers/HomeController.cs [hunk 1]
```diff
@@ -8,6 +9,8 @@ namespace MVC_App.Controllers
 {
     public class HomeController : Controller
     {
+        public readonly testEntities db = new testEntities();
```
Comment (dmyhun-andersen) [inline MVC-App/Controllers/HomeController.cs]: Are you sure it should be _public_?

File: MVC-App/Views/Home/RelationView.cshtml [hunk 2]
```diff
@@ -0,0 +1,28 @@
+﻿
```
Comment (dmyhun-andersen) [inline MVC-App/Views/Home/RelationView.cshtml]: Remove empty line

File: MVC-App/Controllers/HomeController.cs [hunk 3]
```diff
@@ -26,5 +29,21 @@ public ActionResult Contact()
...
+        public ActionResult RelationView()
+        {
+            //using (testEntities db = new testEntities())
+            //{
+            //    //tblRelation relation = db.tblRelation.Where(p => Int32.Parse(p.Id.ToString()) == id).FirstOrDefault();
+            //    return View(db.tblRelation);
+            //}
+            return View(db.tblRelation);
+        }
+
```
Comment (dmyhun-andersen) [inline MVC-App/Controllers/HomeController.cs:42]: If this code in usless just remove it, not comment it.

File: MVC-App/Controllers/HomeController.cs (hunk 1 above)
Comment (WMinowski) [reply MVC-App/Controllers/HomeController.cs]: changed to private

**Review by dmyhun-andersen — APPROVED** [review]
Approved with suggestions provided.

File: MVC-App/Controllers/HomeController.cs (hunk 3 above)
Comment (WMinowski) [reply MVC-App/Controllers/HomeController.cs:42]: this code may be useful in next features, so I kept it

**Pull request merged**